app/(home)/components/MakesDataGrid.tsx

```diff
@@ -14,10 +15,35 @@ const columns: GridColDef[] = [
   { field: "modelsCount", headerName: "# of Models", width: 300 },
 ];
 
+const findMaxNum = (x) => {
+  const mymax = 0;
+  for (let i = 0; i < x.length - 1; i++) {
+    if (x[i] > mymax) {
+      mymax = arr[i];
+    } else {
+      continue;
+    }
+  }
+  return mymax;
+};
+
 export function MakesDataGrid() {
+  const data = [];
+  for (const make of makesAndModels) {
+    data.push(make.models.length);
+  }
+
+  const max = findMaxNum(data);
```

Comment:
```suggestion
  const max = Math.max(makesAndModels.map((make) => make.models.length))
```

why don't we just do this?